cogs/customreactions.py

```diff
@@ -0,0 +1,1111 @@
+                try:
+                    msg_sent_by_user_modifying = False
+                    manual_timeout = time.time() + 60
+                    while not msg_sent_by_user_modifying:
+                        msg = await self.bot.wait_for('message', timeout=60)
+                        if msg.author == user:
+                            response = msg.content
+                            msg_sent_by_user_modifying = True
+                            await msg.delete()
+                        else:
+                            if time.time() > manual_timeout:
+                                raise asyncio.TimeoutError
+
+                except asyncio.TimeoutError:
+                    if proposals:
+                        title = 'The modification of the custom reaction proposal timed out. ' \
+                                'Returning to list of reaction proposals...'
+                    else:
+                        title = 'The modification of the custom reaction timed out. ' \
+                                'Returning to list of current reactions...'
+                    embed = discord.Embed(title=title)
+                    await message.edit(embed=embed)
+                    conn.close()
+                    time.sleep(5)
+                    return
+
+                if response is None:
+                    return
+                elif response.lower() == "stop":
+                    await leave(message)
+                    return
+
+                t = (response, custom_react_id)
+                c.execute(
+                    'UPDATE CustomReactions SET Response = ? WHERE CustomReactionID = ?',
+                    t)
+                conn.commit()
+                self.rebuild_reaction_list()
+                self.rebuild_proposal_list()
+                if proposals:
+                    title = "Response successfully modified! Returning to list of reaction proposals..."
+                else:
+                    title = "Response successfully modified! Returning to list of current reactions..."
+                embed = discord.Embed(title=title)
+                embed.set_footer(text="Modified by {}.".format(user),
+                                 icon_url=user.avatar_url)
+                await message.edit(embed=embed)
+                conn.close()
+                time.sleep(5)
+
+            if reaction.emoji == "3️⃣":
+                embed = discord.Embed(title="Loading...")
+                await message.edit(embed=embed)
+                if proposals:
+                    title = "Modify a custom reaction proposal. React with the option you want"
+                    footer = "{} is currently modifying a custom reaction proposal. \n".format(
+                        user)
+                else:
+                    title = "Modify a custom reaction. React with the option you want"
+                    footer = "{} is currently modifying a custom reaction. \n".format(
+                        user)
```

Comment:
more duplicated string literals – use variables and format as needed

Reply:
See https://github.com/idoneam/Canary/pull/273#issuecomment-656869764